app/components/modals/invite-new-user.js

```diff
@@ -46,6 +46,12 @@ export default ModalComponent.extend(ValidationEngine, {
         this.set('hasValidated', emberA());
     },
 
+    didRender() {
```

Comment:
If there is an easier way (or less hardcoded) to do this than the way I am, let me know. Guessing there is, but this is currently functioning correctly